src/SOS/SOS.UnitTests/Scripts/NestedExceptionTest.script

```diff
@@ -7,19 +7,28 @@
 LOADSOS
 
 # Verify that bpmd works
-IFDEF:PROJECTK
 IFDEF:LIVE
 !IFDEF:MAJOR_RUNTIME_VERSION_1
 
+IFDEF:DESKTOP
```

Comment:
Not across across all the runtimes (2.1 doesn't create a exe, but 3.0/3.1 do).  It would also be some changes to the TestHelper on how/what get launches and it isn't worth all the hassle of getting working after that kind of change.